src/custom-elements/docs/docs.json

```diff
@@ -156,6 +156,7 @@
           "reflectToAttr": true,
           "docs": "to define alert title.",
           "docsTags": [],
+          "default": "false",
```

Comment:
False should be boolean instead of string, or use empty string as default value, but never use "false" string, please.